mailserver.env

```diff
@@ -260,15 +264,7 @@ ENABLE_SPAMASSASSIN=0
 # deliver spam messages in the inbox (eventually tagged using SA_SPAM_SUBJECT)
 SPAMASSASSIN_SPAM_TO_INBOX=1
 
-# KAM is a 3rd party SpamAssassin ruleset, provided by the McGrail Foundation.
-# If SpamAssassin is enabled, KAM can be used in addition to the default ruleset.
-# - **0** => KAM disabled
-# - 1 => KAM enabled
-#
-# Note: only has an effect if `ENABLE_SPAMASSASSIN=1`
-ENABLE_SPAMASSASSIN_KAM=0
```

Comment:
Do I miss something or why was this removed?

Reply:
You missed nothing, I'm an id... this is a nice copy-waste-error... I'm correcting this.

Reply:
correction done in c026d27130c95e787eba0acc4af7e28162bd32b0